polyfills/Element/prototype/matches/config.json

```diff
@@ -10,6 +10,7 @@
 	],
 	"browsers": {
 		"ie": "6 - *",
+		"ie_mob": "10 - *",
```

Comment:
I would prefer it to be two separate PRs if that is possible. To revert the commit you can do this:
```bash
git revert 2b50affe49cd7157bbf67a792480dc2b1ab5f777
```

If you are struggling to do this, I am happy to accept it as one PR :-)